modules/manage/pages/kubernetes/k-scale-redpanda.adoc

```diff
@@ -282,3 +286,11 @@ kubectl get persistentvolumeclaim --namespace <namespace>
 kubectl get persistentvolume --namespace <namespace>
 ----
 
+After the Nodewatcher controller has finished, you have two options to maintain the stability of your Redpanda cluster:
+
+  - *Option 1:* xref:manage:kubernetes/k-decommission-brokers.adoc[Decommission the broker] that was removed from the node. This is necessary to prevent quorum loss and ensure cluster stability.
+
+  - *Option 2:* Recover the node. The StatefulSet will automatically reschedule the broker on the node, maintaining the desired number of brokers in the cluster. This option helps to keep your cluster running at full capacity, but you should still monitor the situation closely and consider decommissioning brokers if node failures persist.
```

Comment:
Recovering the node would be a bit of an involved process and further tricky due to having to deal with the statefulset's ordinal system. The easiest way would be to scale down the STS, manually bind the PV and PVC together, and then decommission the newly added node. We can document that processes but it's involved enough that it should be it's own change and section. WDYT?

Reply:
Tracking here: https://github.com/redpanda-data/documentation-private/issues/2694

Reply:
When this doc is available, we just update this Nodewatcher section to link out to how to recover the broker.